csi/moac/node.js

```diff
@@ -339,16 +339,8 @@ class Node extends EventEmitter {
   async createPool (name, disks) {
     log.debug(`Creating pool "${name}@${this.name}" ...`);
 
-    var poolInfo;
-    try {
-      poolInfo = await this.call('createPool', { name, disks });
```

Comment:
can we not hit other exceptions? Eg if the connection drops?

Reply:
we can and those are caught at higher level in enclosing scope (in poolOperator)

Reply:
:+1: